src/gui/MainApplication.cpp

```diff
@@ -110,11 +110,11 @@ bool MainApplication::winEventFilter(MSG* msg, long* result)
 }
 
 bool MainApplication::nativeEventFilter(const QByteArray& eventType,
-					void* message, long* result)
+					void* message, EventFilter* result)
 {
 	if(eventType == "windows_generic_MSG")
 	{
-		return winEventFilter(static_cast<MSG *>(message), result);
+		return winEventFilter(static_cast<MSG*>(message), reinterpret_cast<long*>(result));
```

Comment:
Instead of using `reinterpret_cast` here, you should change `winEventFilter` to take `EventFilter*` too. Writing to a `qintptr` through a pointer to `long` has undefined behaviour (unless it so happens that they are the same type). `reinterpret_cast` is almost always the wrong choice to fix type errors - changing the types to match is the way to go.

Reply:
Fixed